### PR TITLE
Add BigDL init action

### PR DESCRIPTION
**Pull request by karth295**

### CODE REVIEW

**Comment by karth295** [comment]
Hah this doesn't actually work out of the box on 1.0 (as the README implies). Maybe I should just remove that note and say this only works on 1.2.

**Comment by karth295** [comment]
The latest commits 1) Fix the init action for Dataproc 1.0, 2) update/fix the READMEs.

File: bigdl/bigdl.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+cd /opt/intel-bigdl
+
+wget "${BIGDL_DOWNLOAD_URL}"
+unzip *.zip
+
+JAR=`realpath lib/*.jar`
+PYTHON_ZIP=`realpath lib/*.zip`
+
+echo "SPARK_DIST_CLASSPATH=\"\$SPARK_DIST_CLASSPATH:$JAR\"" >> /etc/spark/conf/spark-env.sh
+echo "PYTHONPATH=\"\$PYTHONPATH:$PYTHON_ZIP\"" >> /etc/spark/conf/spark-env.sh
```
Comment (pmkc) [inline bigdl/bigdl.sh]: nit: cat << EOF >> spark-env.sh is more legible

File: bigdl/bigdl.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# Config changes only need to happen on the master.
+# This way we also avoid DOSing the resource manager when running yarn node -list
+
+if [[ "${ROLE}" == "Master" ]]; then
+  cat conf/spark-bigdl.conf >> /etc/spark/conf/spark-defaults.conf
+  echo "spark.dynamicAllocation.enabled=false" >> /etc/spark/conf/spark-defaults.conf
```
Comment (pmkc) [inline bigdl/bigdl.sh]: You could also combine the spark-defaults into a cat << EOF

**Pull request merged**